docs/getting-started/devenv/using-visual-studio-code.md

```diff
@@ -12,19 +12,387 @@ ms.devlang: dotnet
+to create a new console application.
+
+Your console application depends on the library you built and tested
+in the previous steps. You need to indicate that by editing `project.json`
+to add this dependency.  In the `dependencies` node, add the `Library`
+node below the `Microsoft.NetCore.App` node as follows:
+
+```js
+"dependencies": {
+    "Microsoft.NetCore.App": {
+        "type": "platform",
+        "version": "1.0.0-rc2-3002702"
+    },
+    "library": {
+        "target": "project",
+        "version": "1.0.0-*"
+    }
+}
+```
+
+The `project` node is important here, as it was in the test library. It
+indicates that this is a project in the current solution, and not a
+NuGet package.
+
+Run `dotnet restore` to restore all dependencies. Open `program.cs`
+and replace the contents of the `Main` method with this line:
+
+```cs
+WriteLine($"The answer is {new Thing().Get(19, 23))}");
+```
+
+You'll need to add a couple using directives to the top of the file:
+
+```cs
+using static System.Console;
+using Library;
+```
+
+Then, run `dotnet build`. That creates the assemblies, and you
+can type `dotnet run` to run the executable.
+
+### Debugging your application
+
+You can debug your code in VS Code using the C# extension.
+You install this extension by pressing `F1` to open the VS Code
+palette. Type `ext install` to see the list of extensions. Select the `C#`
+extension. (More details are availble on the [Visual Studio
+Code C# Extension documentation](https://github.com/OmniSharp/omnisharp-vscode/blob/master/debugger.md)
+page.)
+
+After you install the extension, VS Code will ask that you restart the application
+to include the new extension. When you relaunch, VS Code will create a `.vscode` directory
+with two files: `tasks.json` and `launch.json`. These two files control the debugger
+configuration. In most projects, this directory is not included in source control.
+It is included in the sample associated with this walk through so you can see
+the edit you need to make.
+
+Your solution contains multiple projects, so you'll want to modify each of these files
+to perform the correct commands. First, open `tasks.json`. The default build task
+runs `dotnet build` in the workspace source directory. Instead, you want to run it in
+the `src/app` directory. You need to add a `options` node to set the current
+working directory to that:
+
+```js
+"options": {
+    "cwd": "${workspaceRoot}/src/app"
+}
+```
+
+Next, you'll need to open `launch.json` and update the program path. You'll see a
+node under "configurations" that describes the program. You'll see:
+
+```js
+"program": "${workspaceRoot}/bin/Debug/<target-framework>/<project-name.dll>",
+```
+
+You'll change this to:
+
+```js
+"program": "${workspaceRoot}/src/app/bin/Debug/netcoreapp1.0/app.dll",
+```
+
+If you are running on Windows, you'll need to update the Application's `project.json` 
+(in the `src/app` directory) to
```

Comment:
For Windows I had to do this in the library as well or I got an error message when I hit the breakpoint "WARNING: Could not load symbols for 'library.dll'. 'c:\Users\tdykstra\Source\Repos\golden\src\app\bin\Debug\netcoreapp1.0\library.pdb' is a Windows PDB. These are not supported by the cross-platform .NET Core debugger."